addon/background.js

```diff
@@ -373,4 +373,10 @@ async function init() {
   }
 }
 
+// For somewhat unknown reasons we've caused sync probems for some users (probably because of too much data)
+// We've moved from browser.storage.sync to browser.storage.local to fix this.
+// BUT, existing users may still have too much data. This should clear it for them.
+// See also: https://github.com/mozilla/side-view/issues/332 and https://github.com/mozilla/side-view/issues/328
+browser.storage.sync.clear();
```

Comment:
Is this only going to clear side view's data?

Reply:
Yeah, that would be quite a security problem if one add-on could clear another's data!